OpenRA.Game/ModMetadata.cs

```diff
@@ -46,6 +47,9 @@ public class ModMetadata
 				var mod = FieldLoader.Load<ModMetadata>(nd["Metadata"]);
 				mod.Id = m;
 
+				if (nd.ContainsKey("ContentInstaller"))
+					mod.Content = FieldLoader.Load<ContentInstaller>(nd["ContentInstaller"]);
```

Comment:
Actually the `if` should take care of that. I also just reverted locally the addition of `ContentInstaller:` to `modchooser\mod.yaml`, because it's no longer needed (was needed in the earlier stages of this branch).